src/input.ts

```diff
@@ -249,7 +249,7 @@ export class LeanInputAbbreviator {
 
     get active(): boolean {
         const handler = this.handlers.get(window.activeTextEditor);
-        return handler && !!handler.range;
+        return handler && (handler.ranges.length > 0);
```

Comment:
Why not `!!handler.ranges`?  It's eleven characters shorter!  Just kidding, but javascript semantics are fun.